modules/network-operator-cr.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-network-operator.adoc
+// * installing/installing_aws/installing-aws-network-customizations.adoc
+
+[id="network-networkoperator-default-cr-{context}"]
+= Network Operator custom resource
+
+The `Network` configuration in an `operator.openshift.io` Custom Resource (CR)
+object stores the configuration settings for the Network Operator.
+
+The following parameters are set in the `Network.config.openshift.io` CR and
+copied to the `Network.operator.openshift.io` CR by the Network Operator:
+
+* `clusterNetwork.cidr`
+* `clusterNetwork.hostPrefix`
+* `serviceNetwork`
+* `networkType`
+
+The following CR displays the default configuration for the Network Operator and
+explains both the parameters you can configure and valid parameter values:
+
+.Network.operator.openshift.io
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+spec:
+  clusterNetworks:
+  - cidr: 10.128.0.0/14 <1>
+    hostPrefix: 23 <2>
+  serviceNetwork: 172.30.0.0/16 <3>
+  defaultNetwork: <4>
+    type: OpenShiftSDN <5>
+    openshiftSDNConfig: <6>
+      mode: NetworkPolicy <7>
+      mtu: 1450 <8>
+      vxlanPort: 4789 <9>
+  deployKubeProxy: false
+  kubeProxyConfig: <10>
+    bindAddress: 0.0.0.0 <11>
+    iptablesSyncPeriod: 30s <12>
+    proxyArguments:
+      iptables-min-sync-period: <13>
+      - 30s
+----
+
+<1> `cidr`: A block of IP addresses from which pod IPs are allocated.
+{product-title} uses the Classless Inter-Domain Routing (CIDR) standard to
+allocate IP addresses. The 'OpenShiftSDN' network plugin supports multiple
+ClusterNetworks. The address blocks for multiple ClusterNetworks must not
+overlap. This parameter is set in the `config.openshift.io` object.
```

Comment:
Is `config.openshift.io` right through here? Don't you set the value in this CR?

Reply:
The unusual aspect of this particular Operator is that much of its configuration is specified at installation time in `Network.config.openshift.io` and is subsequently made available to the Operator and available in `Network.operator.openshift.io`.